G/glog/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+using BinaryBuilder
+
+name = "Glog"
+version = v"0.4.0"
+
+sources = [
+    ArchiveSource("https://github.com/google/glog/archive/v$(version).tar.gz",
+                  "f28359aeba12f30d73d9e4711ef356dc842886968112162bc73002645139c39c")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/glog-*
+
+mkdir cmake_build
+cd cmake_build/
+
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+      -DCMAKE_TOOLCHAIN_FILE="${CMAKE_TARGET_TOOLCHAIN}" \
+      -DBUILD_SHARED_LIBS=ON \
+      -DBUILD_TESTING=OFF \
+      ..
+
+make -j${nproc}
+make install
+
+# Upstream PR from 30 Aug 2017 "Produce pkg-config file under cmake"
+# See https://github.com/google/glog/pull/239 and #483, not released
+mkdir ${libdir}/pkgconfig
+cat >${libdir}/pkgconfig/libglog.pc <<EOS
```

Comment:
The pkg-config file is always installed in `lib/`
```suggestion
mkdir ${prefix}/lib/pkgconfig
cat >${prefix}/lib/pkgconfig/libglog.pc <<EOS
```